homework.md

```diff
@@ -50,31 +50,33 @@ A programming language used on the internet
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Version Control
+Collaboration
+Issue management
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You deviate from the original
 
 ## Question 10 
 
 What happens when you clone a repostory?
```

Comment:
duplicate it can also means copying it which is something 'Frok" do as well, what we want to know is how they are different from each other. If you want me to elaborate more let me know.

Reply:
Sorry I don't understand what you want sir.

Reply:
now your answer is more clear, well done.